### PR TITLE
tool: yaml format

### PR DESCRIPTION
**Pull request by lovesinatra**
Added a tool that validates/formats YAML

https://github.com/CorentinTh/it-tools/assets/66272034/2c6d2330-3a76-436d-b955-4f296bf2b231

### CODE REVIEW

File: src/tools/yaml-viewer/index.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import { defineTool } from '../tool';
+
+export const tool = defineTool({
+  name: 'YAML prettify and format',
+  path: '/yaml-prettify',
+  description: 'Prettify your YAML string to a human friendly readable format.',
+  keywords: ['yaml', 'viewer', 'prettify', 'format'],
+  component: () => import('./yaml-viewer.vue'),
+  icon: AlignJustified,
+  redirectFrom: ['/yaml-viewer'],
```
Comment (CorentinTh) [inline src/tools/yaml-viewer/index.ts:11]: Redirection is used for legacy routing support 
+ add the creation date 
```suggestion
  createdAt: new Date('2023-11-29'),
```

File: src/tools/yaml-viewer/yaml-models.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { type MaybeRef, get } from '@vueuse/core';
+
+import yaml from 'js-yaml';
```
Comment (CorentinTh) [inline src/tools/yaml-viewer/yaml-models.ts:3]: Maybe use the already installed `yaml` package ?

**Review by CorentinTh — CHANGES_REQUESTED** [review]
Great tool, thank you for the PR ans the video :pray: 
Just few changes and we are good to merge :muscle: 

We currently have the [`yaml`](https://www.npmjs.com/package/yaml) package installed in it-tools and you PR add [`js-yaml`](https://www.npmjs.com/package/js-yaml) (which as not been updated since 3 years)

**Comment by jcgoette** [comment]
What is missing to get this across this finish line?

File: src/tools/yaml-viewer/index.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import { defineTool } from '../tool';
+
+export const tool = defineTool({
+  name: 'YAML prettify and format',
+  path: '/yaml-prettify',
+  description: 'Prettify your YAML string to a human friendly readable format.',
+  keywords: ['yaml', 'viewer', 'prettify', 'format'],
+  component: () => import('./yaml-viewer.vue'),
+  icon: AlignJustified,
+  createdAt: new Date('2023-11-29'),
```
Comment (CorentinTh) [inline src/tools/yaml-viewer/index.ts:11]:
```suggestion
  createdAt: new Date('2024-01-31'),
```

**Review by CorentinTh — DISMISSED** [review]
Hi!
Thank you for this PR :pray: 
And very sorry this very late approval :pleading_face:

**Pull request merged**